C/CGAL/build_tarballs.jl

```diff
@@ -41,19 +36,18 @@ install_license CGAL-*/LICENSE*
 platforms = expand_cxxstring_abis(supported_platforms())
 
 # The products that we will ensure are always built
-products = [
-    LibraryProduct("libCGAL", :libCGAL),
-    LibraryProduct("libCGAL_Core", :libCGAL_Core),
-    LibraryProduct("libCGAL_ImageIO", :libCGAL_ImageIO),
-]
+# CGAL is, as of 5.0, a header-only library, removing support for lib
+# compilation in 5.3
+products = Product[]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    # Essential dependencies
     Dependency("boost_jll"; compat="=1.71.0"),
-    Dependency("GMP_jll", v"6.1.2"),
-    Dependency("MPFR_jll", v"4.0.2"),
+    Dependency("GMP_jll", v"6.1.2"; compat=">=4.2"),
+    Dependency("MPFR_jll", v"4.0.2"; compat=">=2.2.1"),
```

Comment:
We don't have such old versions anyways
```suggestion
    Dependency("GMP_jll"; compat="6.1.2"),
    Dependency("MPFR_jll"; compat="4.0.2"),
```